src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java

```diff
@@ -25,41 +27,55 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
 
         if (player == null)
         {
-            playerMsg(TFM_Command.PLAYER_NOT_FOUND);
+            playerMsg(TFM_Command.PLAYER_NOT_FOUND, ChatColor.RED);
             return true;
         }
 
-        smite(player);
-
-        return true;
-    }
+        String reason = null;
+        if (args.length >= 2)
+        {
+            reason = StringUtils.join(ArrayUtils.subarray(args, 1, args.length), " ");
+        }
 
-    public static void smite(final Player player)
-    {
-        TFM_Util.bcastMsg(player.getName() + " has been a naughty, naughty boy.", ChatColor.RED);
+        TFM_Util.bcastMsg(player.getName() + " has been a smitted for: ", ChatColor.RED);
 
-        //Deop
+        // deop
         player.setOp(false);
 
-        //Set gamemode to survival:
+        // set gamemode to survival:
         player.setGameMode(GameMode.SURVIVAL);
 
-        //Clear inventory:
+        // clear inventory:
         player.getInventory().clear();
 
-        //Strike with lightning effect:
+        // strike with lightning effect:
         final Location targetPos = player.getLocation();
-        final World world = player.getWorld();
         for (int x = -1; x <= 1; x++)
         {
             for (int z = -1; z <= 1; z++)
             {
-                final Location strike_pos = new Location(world, targetPos.getBlockX() + x, targetPos.getBlockY(), targetPos.getBlockZ() + z);
-                world.strikeLightning(strike_pos);
```

Comment:
@OnlineRepz You are not in any position to command anyone to make changes. I highly suggest you refrain from doing something alike from now on.